region/norfair/east/Frog Speedway.json

```diff
@@ -725,6 +729,7 @@
         {"types": ["missiles", "super"], "requires": []},
         {"types": ["powerbomb"], "requires": ["never"]}
       ],
+      "flashSuitChecked": true,
```

Comment:
Added flashSuitChecked to the strats that transition and use screw. I figure the setups and uses can be added later. I could attempt to work on it next, or start to do the easier strats in rooms. 

One thing I noticed is in this strat (speedless speedway) it says you can transition with stored fall speed and deal with the Beetoms with bombs/PBs - I'm pretty sure that youll lose the stored fall speed?

Reply:
Correct, you cannot use bombs or Pbs.  And Screw depends on your amount of stored fall speed.

Reply:
I removed the bomb/PB options. I wasn't sure how to update the screw options if they needed it